src/popup/components/FungibleTokens/TokenDetailsInfo.vue

```diff
@@ -0,0 +1,263 @@
+      </template>
+    </DetailsRow>
+
+    <DetailsRow
+      v-if="poolShare"
+      :label="$t('pages.token-details.poolShare')"
+      :text="poolShare"
+    />
+    <DetailsRow
+      v-if="!tokenData.isAe && UNFINISHED_FEATURES"
+      :label="$t('pages.token-details.transactions')"
+    />
+
+    <DetailsRow
+      v-if="tokenData.total_volume"
+      :label="$t('pages.token-details.volume')"
+    >
+      <template #text>
+        {{ formatCurrency(tokenData.total_volume) }}
+      </template>
+    </DetailsRow>
+    <DetailsRow
+      v-if="tokenData.market_cap_change_24h"
+      class="price"
+      :label="$t('pages.token-details.volumeDaily')"
+    >
+      <template #text>
+        <span
+          :class="{
+            green: tokenData.market_cap_change_percentage_24h > 0,
+            red: tokenData.market_cap_change_percentage_24h < 0,
+          }"
+        >
+          {{ Number(tokenData.market_cap_change_percentage_24h).toFixed(2) }}%
+        </span>
+        {{ formatCurrency(tokenData.market_cap_change_24h) }}
+      </template>
+    </DetailsRow>
+    <DetailsRow
+      v-if="!tokenData.isAe && UNFINISHED_FEATURES"
+      :label="$t('pages.token-details.feeDaily')"
+    />
+
+    <DetailsRow
+      v-if="!tokenData.isAe"
+      class="link"
+      :label="$t('pages.token-details.chart')"
+    >
+      <template #text>
+        <a
+          :href="DEX_URL"
+          target="_blank"
+        >
+          {{ displayDexUrl }}
+          <ExternalLink />
+        </a>
+      </template>
+    </DetailsRow>
+    <DetailsRow
+      v-if="!tokenData.isAe && UNFINISHED_FEATURES"
+      :label="$t('pages.token-details.price-ae')"
+    />
+    <DetailsRow
+      v-if="tokenData.current_price"
+      class="price"
+      :label="$t('pages.token-details.price')"
+    >
+      <template #text>
+        <span
+          :class="{
+            green: tokenData.price_change_percentage_24h > 0,
+            red: tokenData.price_change_percentage_24h < 0,
+          }"
+        >
+          {{ Number(tokenData.price_change_percentage_24h).toFixed(2) }}%
+        </span>
+        {{ formatCurrency(tokenData.current_price) }}
+      </template>
+    </DetailsRow>
+    <DetailsRow
+      v-if="tokenData.ath"
+      :label="$t('pages.token-details.ath-change')"
+      :text="formatCurrency(tokenData.ath)"
+    />
+    <DetailsRow
+      v-if="tokenData.atl"
+      :label="$t('pages.token-details.atl-change')"
+      :text="formatCurrency(tokenData.atl)"
+    />
+  </div>
+</template>
+
+<script>
+import { mapGetters } from 'vuex';
+import BigNumber from 'bignumber.js';
+import DetailsRow from './DetailsRow.vue';
+import AddressShortening from '../AddressShortening.vue';
+import Tokens from '../Tokens.vue';
+import ExternalLink from '../../../icons/external-link.svg?vue-component';
+import { DEX_URL } from '../../utils/constants';
+import { convertToken } from '../../utils/helper';
+
+export default {
```

Comment:
Maybe you could use TypeScript here :)
(only a suggestion, skip it if you want)
Here's an example that you can base of: https://github.com/aeternity/superhero-wallet/blob/refactor/replace-mixin-copy-with-composable/src/popup/components/Modals/TransferReceive.vue